lib/puppet/reports/servicenow.rb

```diff
@@ -8,6 +8,57 @@
   def process
     settings_hash = settings
 
+    case settings['operation_mode']
+    when 'event_management'
+      process_event_management(settings_hash)
+    else
+      # default to incident management
+      process_incident_management(settings_hash)
+    end
+  rescue StandardError => e
```

Comment:
I don't think we need this rescue... puppetserver logs report processor errors anyways. Removing that could be a separate piece of work so can file ticket if people agree?

Reply:
We could ticket it and do the research in the ticket to see what the difference in behavior is. Whichever makes it easier to find and troubleshoot where errors are coming from is fine by me.

Reply:
Filed https://tickets.puppetlabs.com/browse/PIE-340